src/screens/market/top-coins-table/right-swipe-view/right-swipe-view.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { createUseStyles } from '../../../../styles/create-use-styles';
+
+export const useRightSwipeViewStyles = createUseStyles(() => ({
+  rootContainer: {
+    flex: 1,
+    position: 'absolute',
+    right: 0,
+    top: 0,
+    bottom: 0,
```

Comment:
could you use `...StyleSheet.absoluteFillObject` here? it will additionally add `left: 0` btw